flink-python/pyflink/table/tests/test_sql.py

```diff
@@ -58,6 +56,40 @@ def test_sql_query(self):
         expected = ['2,Hi,Hello', '3,Hello,Hello']
         self.assert_equals(actual, expected)
 
+    def test_execute_sql(self):
+        t_env = self.t_env
+        table_result = t_env.execute_sql("create table tbl"
+                                         "("
+                                         "   a bigint,"
+                                         "   b int,"
+                                         "   c varchar"
+                                         ") with ("
+                                         "  'connector' = 'COLLECTION',"
+                                         "   'is-bounded' = 'false'"
+                                         ")")
+        self.assertIsNone(table_result.get_job_client())
+        self.assertIsNotNone(table_result.get_table_schema())
+        self.assertEqual(table_result.get_table_schema().get_field_count(), 1)
+        self.assertIsNotNone(table_result.get_result_kind())
+        self.assertEqual(table_result.get_result_kind(), ResultKind.SUCCESS)
+        table_result.print()
+
+        table_result = t_env.execute_sql("alter table tbl set ('k1' = 'a', 'k2' = 'b')")
+        self.assertIsNone(table_result.get_job_client())
+        self.assertIsNotNone(table_result.get_table_schema())
+        self.assertEqual(table_result.get_table_schema().get_field_count(), 1)
+        self.assertIsNotNone(table_result.get_result_kind())
+        self.assertEqual(table_result.get_result_kind(), ResultKind.SUCCESS)
+        table_result.print()
+
+        table_result = t_env.execute_sql("drop table tbl")
+        self.assertIsNone(table_result.get_job_client())
```

Comment:
The result is always None here. Since Java has not implemented `getJobClient()`, we can:
- Merge this PR after FLINK-16367.  `getJobClient()` has been implemented in this task and it seems the PR will be merged soon.
- Remove the support of `getJobClient()` from the Python side in this PR for now and support it in another task.

Which would you prefer? I would ok with both.

Reply:
I prefer the point that merge this PR after FLINK-16367. getJobClient() has been implemented in this task and it seems the PR will be merged soon.